core-bundle/tests/Image/ImageFactoryTest.php

```diff
@@ -449,7 +458,7 @@ public function testCreatesAnImageObjectFromAnImagePathInLegacyMode(string $mode
         $path = $this->getFixturesDir().'/images/none.jpg';
         $imageMock = $this->createMock(ImageInterface::class);
 
-        $filesystem = $this->createMock(Filesystem::class);
+        $filesystem = $this->getMockBuilder(Filesystem::class)->setMethods(['exists'])->getMock();
```

Comment:
Why is this necessary?

Reply:
Because I don’t want to mock `isAbsolutePath()` because it would returns `null` otherwise.